bin/spiced/src/lib.rs

```diff
@@ -68,7 +73,10 @@ pub struct Args {
 }
 
 pub async fn run(args: Args) -> Result<()> {
-    let app = App::new(".").context(UnableToConstructSpiceAppSnafu)?;
+
+    let current_dir = env::current_dir().unwrap();
```

Comment:
unwrap

Reply:
we could default it to "." if that is allowed by the file watcher

Reply:
Updated, thank you!